ecf/defs/hafs.def

```diff
@@ -0,0 +1,7112 @@
+extern /cycled_v16_3/primary/00/gfs/v16.3/gfs/atmos/analysis/jgfs_atmos_analysis:release_fcst
+extern /cycled_v16_3/primary/18/gfs/v16.3/gfs/atmos/post/jgfs_atmos_post_manager:release_post000
+extern /cycled_v16_3/primary/00/gfs/v16.3/gdas/atmos/post/jgdas_atmos_post_manager:release_post003
+extern /cycled_v16_3/primary/00/gfs/v16.3/gdas/atmos/post/jgdas_atmos_post_manager:release_post006
+extern /cycled_v16_3/primary/00/gfs/v16.3/gdas/atmos/post/jgdas_atmos_post_manager:release_post009
+extern /cycled_v16_3/primary/00/gfs/v16.3/gfs/atmos/post/jgfs_atmos_post_manager:release_post000
+extern /cycled_v16_3/primary/00/gfs/v16.3/gfs/atmos/post/jgfs_atmos_post_manager:release_post126
+extern /cycled_v16_3/primary/18/gfs/v16.3/gfs/atmos/post/jgfs_atmos_post_manager
+
+suite prod_hafs
+  family primary
+    edit hafs_ver 'v1.0.0'
```

Comment:
@lgannoaa Could you please check if hafs_ver should be defined as v1.0 or v1.0.0? Thanks!

Reply:
It should be set to v1.0.0. Thanks

Reply:
@BinLiu-NOAA I modified ecflow workflow definition file with production extern and hafs_ver to v1.0.